go-controller/pkg/config/config.go

```diff
@@ -140,8 +140,8 @@ var (
 	Gateway = GatewayConfig{
 		V4JoinSubnet:       "100.64.0.0/16",
 		V6JoinSubnet:       "fd98::/64",
-		V4MasqueradeSubnet: "169.254.169.0/29",
-		V6MasqueradeSubnet: "fd69::/125",
+		V4MasqueradeSubnet: "169.254.169.0/24",
```

Comment:
does this give use enough of  a range? the limit for UDNs right now is over 4k IIRC so we would easily exhaust this range.

Reply:
> does this give use enough of a range? the limit for UDNs right now is over 4k IIRC so we would easily exhaust this range.

I'm just increasing it a lil' bit so it can work.

The PR that will increase the subnet can be found in: https://github.com/ovn-org/ovn-kubernetes/pull/4496/files

Maybe I should rebase it.

Reply:
Done. Dropped this commit.